crates/runtime-factors/src/lib.rs

```diff
@@ -100,6 +100,10 @@ pub struct TriggerAppArgs {
     /// To run from a file, prefix the filename with @ e.g. spin up --sqlite @migration.sql
     #[clap(long = "sqlite")]
     pub sqlite_statements: Vec<String>,
+
+    // Sets the maxmimum memory allocation limit for an instance.
+    #[clap(long = "max-memory-size", env = "MAX_MEMORY_SIZE")]
```

Comment:
I think we prefix env vars elsewhere

```suggestion
    #[clap(long = "max-memory-size", env = "SPIN_MAX_MEMORY_SIZE")]
```